tests/core/test_listener.py

```diff
@@ -57,3 +57,6 @@ def test_listener_has_default_impl_for_mute_changed(self):
 
     def test_listener_has_default_impl_for_seeked(self):
         self.listener.seeked(0)
+
+    def test_listener_has_default_impl_for_current_metadata_changed(self):
```

Comment:
Heh, this will likely break with my other changes I have underway, guess we have to see who of us can get the change in first :P